### DISCUSSION TITLE
The scale increases if you do not touch the image when cropping

### DISCUSSION THREAD

**Discussion by proxymades** [ValentinH/react-easy-crop#533] [Q&A]
Here is the image cropped - here I moved it and clicked crop
<img width="369" alt="Screenshot 2024-04-26 at 11 52 48" src="https://github.com/ValentinH/react-easy-crop/assets/81700804/555d5862-7ebd-45dd-a029-192dcc687f49">

Here is the image cropped and automatically enlarged to maximum - here I didn’t touch anything and just clicked crop
<img width="370" alt="Screenshot 2024-04-26 at 11 53 17" src="https://github.com/ValentinH/react-easy-crop/assets/81700804/fac296e8-8778-4045-a662-6661e13252cf">

What could be the problem? I set maxZoom to 1, this doesn’t help either

**Comment by ValentinH** [comment]
This seems to be an initial state not being properly set issue.
Without code it's hard for me to understand what is going on.
